src/lib/components/healthselectorv2/HealthSelector.component.js

```diff
@@ -0,0 +1,121 @@
+//@flow
+import * as React from 'react';
+import { components } from 'react-select';
+import type { OptionProps } from '../selectv2/Selectv2.component';
+import type { SelectProps } from '../selectv2/Selectv2.component';
+import {
+  SingleValueWrapper,
+  ValueIcon,
+  OptionIcon,
+  OptionWrapper,
+  OptionLabel,
+  SelectStyle,
+  ShortLabel,
+  icons,
+} from './HealthSelector.style';
+
+type optionType = 'all' | 'healthy' | 'warning' | 'critical' | 'unknown';
+type optionValue = {
```

Comment:
nit: Case of typings
```suggestion
type OptionType = 'all' | 'healthy' | 'warning' | 'critical' | 'unknown';
type OptionValue = {
```